pipelines/controller/src/main/java/org/openmrs/analytics/PipelineManager.java

```diff
@@ -78,6 +86,41 @@ public class PipelineManager implements ApplicationListener<ApplicationReadyEven
   // TODO expose this in the web-UI
   private LastRunStatus lastRunStatus = LastRunStatus.NOT_RUN;
 
+  /**
+   * This method publishes the beam pipeline metrics to the spring boot actuator. Previous metrics
+   * are removed from the actuator before publishing the latest metrics
+   *
+   * @param metricResults
+   */
+  void publishPipelineMetrics(MetricResults metricResults) {
+    removePipelineMetrics();
+    MetricQueryResults metricQueryResults = EtlUtils.getMetrics(metricResults);
+    for (MetricResult<Long> metricCounterResult : metricQueryResults.getCounters()) {
+      Counter counter =
+          meterRegistry.counter(
```

Comment:
Do we show any description (e.g., like comments in your PR snapshot) for these counters? If not, can we please add some?

Reply:
HELP and TYPE sections are auto populated by default. Please have a look at the latest snapshot in the PR description.

Reply:
Okay thanks; we can resolve this.